docs/faq.rst

```diff
@@ -712,18 +712,28 @@ will make the subsequent rebuilds faster (because it needs to transfer less data
 from the repository). While being faster, the cache needs quite some disk space,
 which might be unwanted.
 
-There is a temporary (but maybe long lived) hack to avoid using lots of disk
-space for chunks.archive.d (see :issue:`235` for details):
+A temporary (but maybe long lived) hack to avoid using lots of disk memory
+for ``chunks.archive.d`` (see :issue:`235` for details) is to replace this
+directory with an empty dummy file of this name.  Note, below,
+``$(borg config id)`` is used as place holder for the alphanumeric string
+specific to your backup repository (REPO_ID) you have to enter instead.
+You identify this string of 64 characters either in your file browsers, or
```

Comment:
The usual term is **file manager**.

Reply:
So this was influenced by [Apple advertisement](https://apps.apple.com/us/app/filebrowser-document-manager/id364738545) [English Wikipedia](https://en.wikipedia.org/wiki/File_manager) noting both forms; then sticking to the proper purpose.  Your point is convincing, file manager.